design-system-docs/bridgetown.config.yml

```diff
@@ -0,0 +1,31 @@
+# Welcome to Bridgetown!
+#
+# This config file is for settings that affect your whole site, values
+# which you are expected to set up once and rarely edit after that.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# `bin/bridgetown start`. If you change this file, please restart the server process.
+#
+# For reloadable site metadata like title, SEO description, social media
+# handles, etc., take a look at `src/_data/site_metadata.yml`
+#
+# If you need help with YAML syntax, here are some quick references for you:
+# https://learn-the-web.algonquindesign.ca/topics/markdown-yaml-cheat-sheet/#yaml
+# https://learnxinyminutes.com/docs/yaml/
+#
+
+url: '' # the base hostname & protocol for your site, e.g. https://example.com
+
+permalink: pretty
+
+template_engine: erb
+
+collections:
```

Comment:
Think this makes sense for now, but do we have to have the `cads_` prefix in URLs? Is it work trying out the new Resources feature as we're already running the 1.0 alpha https://www.bridgetownrb.com/docs/resources

Reply:
Yeah probs - there is already a collection called `components` that comes with the installation, and it's where bridgetown expects you to create your custom view components or whatevs, so I just prefixed to get the ball rolling.  Once for the ideation sesh, for sure.

Reply:
Ah! Yeah that rings a bell. Cool, let's leave it like this for now.